root/report/KaraokeRecordingsMarkedInstrumental.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2022 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import RecordingList from './components/RecordingList';
+import ReportLayout from './components/ReportLayout';
+import type {ReportDataT, ReportRecordingT} from './types';
+
+
+const KaraokeRecordingsMarkedInstrumental = ({
+  canBeFiltered,
+  filtered,
+  generated,
+  items,
+  pager,
+}: ReportDataT<ReportRecordingT>): React.Element<typeof ReportLayout> => (
+  <ReportLayout
+    canBeFiltered={canBeFiltered}
+    description={l(
+      `This report shows recordings which have a “karaoke” relationship to
+       another recording, but are also linked to a work with the
+       “instrumental” attribute. Since lyrics are relevant to karaoke
+       recordings, “instrumental” should not be used on them. Alternatively,
+       if this is not a karaoke recording, but just an instrumental, it
+       shouldn’t be linked to another recording with a karaoke relationship.`,
+    )}
```

Comment:
It isn’t clear how an instrumental with lyrics should be handled. I started this discussion in [comment to the ticket](https://tickets.metabrainz.org/browse/MBS-11916?focusedCommentId=63224&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-63224) so as to include issue’s reporter too.

Reply:
That's a huge edge case, it's not clear how it should be handled because there's been no discussion before (I've never seen the concept), but it doesn't seem problematic for all the clear-cut cases :)

Reply:
It helped to narrow the issue: The terms “instrumental” and “karaoke” are widely used outside of MusicBrainz in different ways. It seems it would be worth reminding in the report’s description that these terms have a specific meaning in the context of MusicBrainz.

Reply:
Added `extraInfo`, do check if that seems better :)

Reply:
It is slightly better, but still confusing, especially the undefined “standard instrumental”. I failed to improve the wording though, likely because the definition of “instrumental” in MusicBrainz is needlessly too far from the common definition. Both attributes “karaoke” and “instrumental” could be allowed at the same time if the definition of “instrumental” was broadened to match linking recordings without vocals to works with lyrics. It would more be a STYLE issue though.

Reply:
Can this link to the guidelines where karaoke and instrumental are defined?

Reply:
I did that now.